crates/actor/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+wit_bindgen::generate!({
+    path: "../../wit",
+    world: "wasmcloud.interfaces",
+});
+
+#[cfg(feature = "rand")]
+pub use rand::{Rng, RngCore};
```

Comment:
Should we be using `use cap_rand::{Rng, RngCore};` ?

https://github.com/bytecodealliance/wasmtime/pull/5244/files